packages/shared/lib/core/network/utils/isEvmChain.ts

```diff
@@ -0,0 +1,6 @@
+import { IEvmChainConfiguration } from '../interfaces'
+import { ChainConfiguration } from '../types'
+
+export function isEvmChain(configuration: ChainConfiguration): configuration is IEvmChainConfiguration {
```

Comment:
what exactly does `configuration is IEvmChainConfiguration` do?

Reply:
It's a type guard. It ensures that the type of `configuration` is narrowed down to `IEvmChainConfiguration`